c/olc_lookup.h

```diff
@@ -0,0 +1,24 @@
+// THIS FILE WAS GENERATED AUTOMATICALLY
+// DO NOT EDIT IT UNLESS YOU KNOW WHAT YOU ARE DOING
+
+static int kAlphabetPositionLUT[256] = {
```

Comment:
Can you generate a comment that explains what this is? It's not obvious looking at it what it represents and what each row/column represents.

Reply:
Added with commit 3f786403e5b620f50f5544c7bf6ea0a8d4982f04